### ISSUE TITLE
Tutorial Text

### ISSUE THREAD

**Issue by RamsesCaesar** [RamsesCaesar/CatAmongBirds#36]
There should be some text that appears and tells the player how to move, swat, and run. Also how to gain hearts.

**Comment by RamsesCaesar** [comment]
There is some text that tells the player the game controls at game start now.
